### ISSUE TITLE
Changing toggles in Mod Integration config do not affect checks in UTMixinLoader

### ISSUE THREAD

**Issue by jchung01** [ACGaming/UniversalTweaks#365]
As title says, if you change a toggle option in the Mod Integration config, it does not seem to be reflected in the checks in `UTMixinLoader`, so mixins are applied regardless of the checks, only depending on the default values. If the mixin classes itself have the checks, those still work as they are an extra safeguard. 

As an example, let's add Blood Magic and consider `utDuplicationFixesToggle`. This is default `true`, but setting it to `false` in the config will still queue `mixins.mods.bloodmagic.dupes.json` even though the config option should be checked in `UTMixinLoader`. This means regardless of the duplication fixes toggle option, the dupe fix mixin will be applied. This can be checked through the debug log or by checking exported mixins after opening the Sigil of Holding in this specific case.

In other words, it seems that config values aren't read from file by the time `UTMixinLoader` checks the config. This doesn't seem to be a problem for the other types of mixins loaded by `UTLoadingPlugin` though. I don't know if this is an issue due to `ILateMixinLoader` or using ConfigAnytime.

**Comment by Rongmario** [comment]
Please try it with this build of ConfigAnytime: [!configanytime-1.0-INTERNAL.zip](https://github.com/ACGaming/UniversalTweaks/files/14101882/configanytime-1.0-INTERNAL.zip).

(unzip before adding it to mods folder, since github doesn't allow .jar files, it is packed inside a .zip)

**Comment by ACGaming** [comment]
Positively tested with the mentioned Blood Magic config!

**Comment by ACGaming** [comment]
Fixed with CleanroomMC/ConfigAnytime@0d59e37c45cfdb822ad0b9a08771b4e47f53f154
